assignments/objects.js

```diff
@@ -49,16 +85,37 @@ const example = {
 // 3. Nest a grandchild object in the child object with properties for name and age.  The name will be Sam and the age will be 30
 // 4. Give each of the objects the ability to speak their names using the this keyword.
 
-const parent = {}
```

Comment:
This is a solid mastery of understanding objects, especially when you nest it and be able to call it properly like CSS.